app/components/svg_component.rb

```diff
@@ -2,6 +2,10 @@ class SvgComponent < ApplicationComponent
   # { symbol: method returning path for symbol }
   ICON_MAPPINGS = {
     cart: :cart,
+    # @todo I don't know where we have been getting the SVG icons;
+    #       and I am hesitant to add one unless it's coming from
+    #       a library we are already using... - ZS
+    money: :cart,
```

Comment:
@KellyAH or @anaulin - you both have found SVG icons; do you have a site you recommend with generous licensing?